promql/functions.go

```diff
@@ -953,33 +952,20 @@ func funcStddevOverTime(vals []parser.Value, args parser.Expressions, enh *EvalN
 			mean, cMean = kahanSumInc(delta/count, mean, cMean)
 			aux, cAux = kahanSumInc(delta*(f.F-(mean+cMean)), aux, cAux)
 		}
-		return math.Sqrt((aux + cAux) / count)
+		return varianceToResult((aux + cAux) / count)
```

Comment:
To simplify, and possibly optimize (in case the extra function call makes a difference), how about making `varianceToResult` optional?
```suggestion
		variance := (aux + cAux) / count
		if varianceToResult == nil {
			return variance
		}
		return varianceToResult(variance)
```

Reply:
Yeah, that would be a good improvement. I'll implement it soon.